api/src/main/java/javax/servlet/ServletContext.java

```diff
@@ -541,8 +541,8 @@
      * The attribute is returned as a <code>java.lang.Object</code> or some subclass.
      *
      * <p>
-     * Attribute names should follow the same convention as package names. The Java Servlet API specification reserves
-     * names matching <code>java.*</code>, <code>javax.*</code>, and <code>sun.*</code>.
+     * Attribute names should follow the same convention as package names. The Jakarta Servlet specification reserves
+     * names matching <code>jakarta.*</code>, <code>java.*</code>, <code>javax.*</code>, and <code>sun.*</code>.
```

Comment:
I think it is important that we reserve the `jakarta.*` name space ahead of the actual package rename.    
I guess it is OK that we continue to reserve `java.*` and `javax.*`?

Reply:
You can't add new requirements to this version of the spec.
This will have to be added for Jakarta EE 9.

Reply:
ack